### PR TITLE
Yc 384

### PR DESCRIPTION
**Pull request by nicolas-sitylb**
Fix # 384 in Jira.

### CODE REVIEW

File: permits/admin.py [hunk 1]
```diff
@@ -767,6 +840,14 @@ def __new__(cls, *args, **kwargs):
 
 
 class PermitAdministrativeEntityAdminForm(forms.ModelForm):
+    """ Form class to configure an administrative entity (commune, organisation)
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(PermitAdministrativeEntityAdminForm, self).__init__(*args, **kwargs)
+        self.fields["expeditor_name"].disabled = False
+        self.fields["expeditor_name"].initial = None
```
Comment (monodo) [inline permits/admin.py:849]: useful ? if not, please remove

**Review by monodo — CHANGES_REQUESTED** [review]

File: permits/admin.py (hunk 1 above)
Comment (nicolas-sitylb) [reply permits/admin.py]: Done.

File: permits/admin.py [hunk 2]
```diff
@@ -223,6 +228,17 @@ def get_queryset(self, request):
...
         return qs
 
+    def save_model(self, req, obj, form, change):
+        """ Set 'is_staff=True' when the saved user is in a integrator group.
+        """
+        if req.user.is_superuser:
+            obj.is_staff = False
+            for group in form.cleaned_data["groups"]:
+                if group.permitdepartment.is_integrator_admin:
+                    obj.is_staff = True
```
Comment (monodo) [inline permits/admin.py:238]: If `obj.is_superadmin`, do nothing. Otherwise, superadmin could loose it's admin access

File: permits/admin.py [hunk 3]
```diff
@@ -364,11 +380,45 @@ def clean_permissions(self):
...
     inlines = (PermitDepartmentInline,)
     form = GroupAdminForm
+    list_display = [
+        "__str__",
+        "get__is_validator",
+        "get__is_default_validator",
+        "get__integrator",
+        "get__mandatory_2fa",
+    ]
+
```
Comment (monodo) [inline permits/admin.py:390]: If not too complicated, change the display to checkbox like in WOTs for instance (1.4)

image

**Review by monodo — CHANGES_REQUESTED** [review]
2 little points to check/fix

File: permits/admin.py (hunk 2 above)
Comment (monodo) [reply permits/admin.py:238]: https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser

Comment (nicolas-sitylb) [reply permits/admin.py:239]: Resolved by https://github.com/yverdon/geocity/pull/383/commits/7a2f14fd1dac30ea60bd329856740907910bfe15

**Comment by nicolas-sitylb** [comment]
Fixed `is_staff` for super users + synchronized with the `main` branch.    
(also added an `order` field in the WorksObject model which could serve as a basis to order the works objects).

**Review by monodo — APPROVED** [review]

**Pull request merged**